embedded-kafka/src/main/scala/io/github/embeddedkafka/ops/kafkaOps.scala

```diff
@@ -240,13 +241,15 @@ trait RunningKafkaOps {
 
   private[embeddedkafka] def kafkaPort(kafkaBrokerServer: BrokerServer): Int =
     kafkaBrokerServer.boundPort(
-      kafkaBrokerServer.config.listeners.head.listenerName
+      new ListenerName(kafkaBrokerServer.config.listeners.head.listener())
```

Comment:
From what I've seen in https://github.com/apache/kafka/pull/19191, we should use this to be safe:

```suggestion
      ListenerName.normalised(kafkaBrokerServer.config.listeners.head.listener())
```